tgui/packages/tgui/interfaces/RestockTracker.jsx

```diff
@@ -81,7 +81,17 @@ export const RestockTracker = (props) => {
             </Stack.Item>
           </Stack>
         ))}
+        {vending_list.length ? null : <RestockTrackerFull />}
```

Comment:
```suggestion
        {vending_list.length === 0 && <RestockTrackerFull />}
```
explicit length check + no need to tell react to show null